### PR TITLE
fix: account change test broken on beta testnet

### PR DESCRIPTION
**Pull request by jstarry**

### CODE REVIEW

File: src/connection.js [hunk 1]
```diff
@@ -399,9 +399,10 @@ const GetVoteAccounts = jsonRpcResult(
         nodePubkey: 'string',
         activatedStake: 'number',
         epochVoteAccount: 'boolean',
-        epochCredits: struct.union(['null', struct.list([
-          struct.tuple(['number', 'number', 'number']),
-        ])]),
+        epochCredits: struct.union([
+          'null',
```
Comment (mvines) [inline src/connection.js]: epochCredits landed in beta, https://github.com/solana-labs/solana/pull/7445, so why do we need this?

Comment (jstarry) [reply src/connection.js]: I thought @jackcmay needed v0.21 compatibility but maybe not?

Comment (mvines) [reply src/connection.js]: Yes he certainly does.  But epochCredits is now in v0.21

Comment (jackcmay) [reply src/connection.js]: that fix also fixed one of the two failures I was seeing with the beta localnet

Comment (jstarry) [reply src/connection.js]: > But epochCredits is now in v0.21


Ah ok, good point, I'll remove.

Comment (jstarry) [reply src/connection.js]: removed here: https://github.com/solana-labs/solana-web3.js/pull/577

**Pull request merged**